lib/beaker/ca_cert_helper.rb

```diff
@@ -0,0 +1,129 @@
+require 'beaker'
+require 'beaker/dsl/helpers/host_helpers'
+
+module Beaker::CaCertHelper
+
+  def install_ca_certs
+    install_ca_certs_on(hosts)
+  end
+
+
+  ##
+  # Install certs on Windows and OSX Hosts
+  #
+  # @param Array<Host>
+  #
+  ##
+  def install_ca_certs_on(hosts)
+    Array[hosts].each do |host|
+      get_cert_hash.each do |cert_name, ca|
+        create_cert_on_host(host, cert_name, ca)
+        if host['platform'] =~ /windows/i
+          add_windows_cert host, cert_name
+        end
+      end
+    end
+  end
+
+  ##
+  # create the remote file and ensure proper rights
+  #
+  def create_cert_on_host(host, cert_name, ca)
+    create_remote_file(host, cert_name, ca)
+    on host, "chmod 644 #{cert_name}"
+  end
+
+  ##
+  # Install cert, assumes cygwin currently
+  ##
+  def add_windows_cert(host, ca_filename)
+    on host, "cmd /c certutil -v -addstore Root `cygpath -w #{ca_filename}`"
```

Comment:
I think it would be better to not reference cygwin here. Couldn't the `install_ca_certs_on` method copy the cert to a windows path, e.g. `%SYSTEMDRIVE%%SYSTEMROOT%\temp\#{cert_name}` on the target node (or use one of the beaker methods that create a file with contents and return a "native" file path), and then call `add_windows_cert` with the windows path?